geth-dev-node/start.sh

```diff
@@ -1,3 +1,2 @@
-geth --password /app/password.txt account new
-geth init /app/genesis.json
-geth --password /app/password.txt --unlock 0 --mine --networkid=10000 --ws --wsapi eth,net,web3,personal --wsport 8546 --rpc --rpcapi eth,net,web3,personal,miner --rpcaddr 0.0.0.0 --targetgaslimit 6700000 --ethash.dagdir "/app/.ethash"
+geth --datadir /home/\.ethereum init /home/genesis.json
+geth --datadir /home/\.ethereum --keystore /home/\.ethereum/keystore --password /home/password.txt --unlock "0xc5ed899b0878656feb06467e2e9ede3ae73cbcb7" --mine --ws --wsapi eth,net,web3,personal --wsport 8546 --rpc --rpcapi eth,net,web3,personal,miner --rpcaddr 0.0.0.0 --targetgaslimit 6500000
```

Comment:
Is `--mine` still necessary even though this is a PoA network?

Reply:
Good question.  I'll try taking it out, and if everything still works, I'll leave it out.

Reply:
Yep, looks like it is necessary.  Nothing happens after unlocking the account without `--mine`.

Reply:
Interesting.  It must use `--mine` to mean `validate` when running in Proof of Authority mode.